gui/wxpython/location_wizard/wizard.py

```diff
@@ -179,6 +179,11 @@ def __init__(self, wizard, parent, grassdatabase):
                 grass.legal_name,
                 self._nameValidationFailed))
         self.tlocTitle = self.MakeTextCtrl(size=(400, -1))
+        
+        # text for required options
+        self.required_txt = self.MakeStaticText("*", size=(-1, -1))
```

Comment:
I don't think the required_txt needs to be attribute of the class (remove self).